tests/compose/fixtures/extends/common.yml

```diff
@@ -0,0 +1,7 @@
+web:
+  image: busybox
+  command: /bin/true
+  net: host
```

Comment:
On vic, we don't have the `host` network. We only have `bridge` and use-defined networks (if any).